File: FrontEnd/SoftPac/SoftPacPersistance/DAO/DetallesFacturaDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface DetallesFacturaDAO
    {
        int Insertar(DetallesFacturaDTO detalleFactura);
        int Modificar(DetallesFacturaDTO detalleFactura);
        int Eliminar(DetallesFacturaDTO detalleFactura);
        int EliminarLogico(DetallesFacturaDTO detalleFactura);
        DetallesFacturaDTO ObtenerPorId(int detalleFacturaId);
        IList<DetallesFacturaDTO> ListarTodos();
    }
}

File: BackEnd/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/DetallesFacturaDAO.java
package pe.edu.pucp.softpac.dao;

import pe.edu.pucp.softpac.model.DetallesFacturaDTO;
import pe.edu.pucp.softpac.model.UsuariosDTO;

public interface DetallesFacturaDAO {
    public Integer insertar(DetallesFacturaDTO detallePropuesta);

    public DetallesFacturaDTO obtenerPorId(Integer detallePropuestaId);

    public Integer modificar(DetallesFacturaDTO detallePropuesta);

    public Integer eliminarPorFactura(Integer facturaId);

    public Integer eliminarLogico(DetallesFacturaDTO detalleFactura);
}
